src/User/ProfilePage.jsx

```diff
@@ -14,19 +29,19 @@ const ProfilePage = ({ user, isLoading, error }) => {
     return <p>An error has occurred...</p>;
   }
 
-  const { username, firstName, lastName, email, occupation } = user || {};
+  const { id, username, firstName, lastName, email, occupation } = user || {};
 
   return (
-    <StyledForm>
-      <StyledCard>
-        <StyledCardHeading className="name">
-          Hello {firstName} {lastName}
-        </StyledCardHeading>
-        <p className="username">Username: {username}</p>
-        <p className="email">Email: {email}</p>
-        <p className="occupation">Occupation: {occupation}</p>
-      </StyledCard>
-    </StyledForm>
+    <StyledCard>
+      <StyledLink className="editLink" to={`/users/edit/${id}`}>Edit</StyledLink>        
```

Comment:
You may want to check if the current users Id matches the person you are viewing in order to show this button